.github/workflows/unit.yml

```diff
@@ -27,11 +27,11 @@ jobs:
       matrix:
         include:
           - type: ethereum
-            args: --features "asm-keccak ethereum" --locked
+            args: --features "asm-keccak ethereum rayon" --locked
             partition: 1
             total_partitions: 2
           - type: ethereum
-            args: --features "asm-keccak ethereum" --locked
+            args: --features "asm-keccak ethereum rayon" --locked
```

Comment:
these shouldn't be necessary we want to enable this always for the node itself.

I think we want this enabled in the node-builder crate so it will be enabled for all binaries